### DISCUSSION TITLE
Welcome to WP Multisite WaaS Discussions!

### DISCUSSION THREAD

**Discussion by superdav42** [Ultimate-Multisite/ultimate-multisite#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Really want to see what interest there is in maintaining this plugin.

**Comment by mredodos** [comment]
You can count on me if you need help on coding, web and what you need :) 

Im Edoardo im born in rome but i live in Zurich(Switzerland) ima fullstack developer in PHP and JS vanilla, now im working a lot with python, react, and a lot of other stuff :)

Nice to meet you!

![image](https://github.com/user-attachments/assets/f01d74c1-7240-4af6-9db1-8521d0ea7fc4)

**Reply by aniclip1** [reply]
If you can update the free plugin it will be nice :
[https://github.com/superlatif-io/wpultimo-plesk-gateway](url)

**Reply by twicealive** [reply]
Are you available to work on this plugin?  WP Multisite WaaS

**Comment by aniclip1** [comment]
Hello David Stone,
Thank you very much to make a reborn of this magical plugin.
I use it since 2020, I quit it on july 24 but you relaunch my interest and I install it 2 hours ago :-) 
A question about these add-on :
material-wp.zip
pro-theme.zip
wp-ultimo-admin-page-creator.zip
wp-ultimo-ads.zip
wp-ultimo-affiliatewp.zip
wp-ultimo-captcha.zip
wp-ultimo-language-selector.zip
wp-ultimo-mailchimp.zip
wp-ultimo-multiple-accounts.zip
wp-ultimo-plugin-and-theme-manager.zip
wp-ultimo-pro-sites-migrator.zip
wp-ultimo-site-exporter.zip
wp-ultimo-support-agents.zip
wp-ultimo-vat.zip
wp-ultimo-woocommerce.zip

Do you still use it ? Can I install it without issue ?
Thanks for your reply

**Reply by TMJr85** [reply]
Hi there, could you please share those plugins?

**Reply by aniclip1** [reply]
some give fatal error after test ...

**Reply by superdav42** [reply]
All of these add-on can be found in [gdrive](https://drive.google.com/drive/folders/19gYtCd49UwFdJy5pWKRJjyWSAYyZi4F4?usp=sharing) for now. They should work. Please open a detailed github issue if you have any problems. Like what the error is, versions used. steps to reproduce and screenshots.

**Reply by marcusquinn** [reply]
I believe some of the addons got rolled into the core plugin plugin at some point, so just check to see if the features already exist in the latest version, as you may already have the replacement for the addon.

**Comment by marcusquinn** [comment]
Here for the good times. Fan of WP & WP Multisite. Much easier to manage many sites when you only have to update plugins once, and can work in the same database for many networked sites.
